Relax tolerance for tests/linalg_test.py::ScipyLinalgTest::testLu_shape=float64[50,50]
which failes by a small amount on AMD Rome
Patch by Simon Branford (University of Birmingham)
--- tests/linalg_test.py.orig	2022-06-21 19:05:51.211914000 +0100
+++ tests/linalg_test.py	2022-06-21 19:10:57.704763179 +0100
@@ -1031,8 +1031,9 @@
     x, = args_maker()
     p, l, u = jsp.linalg.lu(x)
     self.assertAllClose(x, np.matmul(p, np.matmul(l, u)),
-                        rtol={np.float32: 1e-3, np.float64: 1e-12,
-                              np.complex64: 1e-3, np.complex128: 1e-12})
+                        rtol={np.float32: 1e-3, np.float64: 1e-11,
+                              np.complex64: 1e-3, np.complex128: 1e-12},
+                        atol={np.float64: 1e-14})
     self._CompileAndCheck(jsp.linalg.lu, args_maker)
 
   def testLuOfSingularMatrix(self):
